### PR TITLE
chore(books-es): complete resource creators

### PR DESCRIPTION
**Pull request by davorpa**
## What does this PR do?
Add info | Improve repo

### Why is this valuable (or not)?

It completes the name of book-type spanish resources there where don't already have it.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-es.md [hunk 1]
```diff
@@ -54,24 +54,24 @@
 
 * [Aprender Python](https://wiki.python.org.ar/aprendiendopython/) - Python Argentina
 * [Apuntes Completos de Desarrollo Web](http://jorgesanchez.net) - Jorge Sánchez
-* [Asombroso DDD: Una lista curada de recursos sobre Domain Driven Design](https://github.com/ddd-espanol/asombroso-ddd)
+* [Asombroso DDD: Una lista curada de recursos sobre Domain Driven Design](https://github.com/ddd-espanol/asombroso-ddd) - Domain Driven Design en Español
```
Comment (eshellman) [inline books/free-programming-books-es.md:57]: I don't think the added entity is useful

File: books/free-programming-books-es.md [hunk 2]
```diff
@@ -54,24 +54,24 @@
...
 * [Desarrollo de Aplicaciones Web - Temario Completo](https://github.com/statickidz/TemarioDAW#temario-daw) - José Luis Comesaña (GitHub)
 * [Desarrollo de Aplicaciones Web y Sistemas Microinformáticos y Redes](https://javiergarciaescobedo.es) - Javier García Escobedo
-* [Gitbook - Libros útiles en español](https://github.com/DpdC/gitbook-biblioteca-impresionante-en-espanol#gitbook---biblioteca-de-libros-en-español--) (GitHub)
-* [Múltiples Cursos y Enlaces de Tecnología Informática](http://elvex.ugr.es) - Fernando Berzal
-* [OpenLibra - Biblioteca recopilatorio de libros libres](https://openlibra.com/es/collection)
+* [Gitbook - Libros útiles en español](https://github.com/rosepac/gitbook-biblioteca-impresionante-en-espanol#gitbook---gu%C3%ADas-creadas-en-gitbook-en-espa%C3%B1ol) - Pablo Alvarez Corredera (GitHub)
+* [Múltiples Cursos y Enlaces de Tecnología Informática](http://elvex.ugr.es) - Fernando Berzal Galiano
+* [OpenLibra - Biblioteca recopilatorio de libros libres](https://openlibra.com/es/collection) - OpenLibra
```
Comment (eshellman) [inline books/free-programming-books-es.md:62]: repetitive - remove

File: books/free-programming-books-es.md [hunk 3]
```diff
@@ -161,53 +161,53 @@
 ### Go
 
 * [El pequeño libro Go](https://raulexposito.com/the-little-go-book-en-castellano.html) - Karl Seguin, `trl.:` Raúl Expósito (HTML, [PDF](https://raulexposito.com/assets/pdf/go.pdf), [EPUB](https://raulexposito.com/assets/epub/go.epub))
-* [Go en Español](https://nachopacheco.gitbooks.io/go-es/content/doc) - Nacho Pacheco (HTML)
+* [Go en Español](https://nachopacheco.gitbooks.io/go-es/content/doc) - Nacho Pacheco (HTML) (GitBook)
```
Comment (eshellman) [inline books/free-programming-books-es.md:164]: we don't list platform for books

File: books/free-programming-books-es.md [hunk 4]
```diff
@@ -161,53 +161,53 @@
...
+* [Ejercicios de Programación en Java](https://www.arkaitzgarro.com/java/) - Arkaitz Garro, Javier Eguíluz Pérez, A. García-Beltrán, J.M. Arranz (PDF)
 * [Notas de Introducción al Lenguaje de Programación Java (2004)](https://lya.fciencias.unam.mx/jloa/publicaciones/introduccionJava.pdf) - Universidad Nacional Autónoma de México, Jorge L. Ortega Arjona (PDF)
 * [Pensando la computación como un científico (con Java)](http://www.ungs.edu.ar/cm/uploaded_files/publicaciones/476_cid03-Pensar%20la%20computacion.pdf) (PDF)
-* [PlugIn Apache Tapestry: desarrollo de aplicaciones y páginas web](https://picodotdev.github.io/blog-bitix/assets/custom/PlugInTapestry.pdf) (PDF) [(ePub, Mobi, HTML)](https://picodotdev.github.io/blog-bitix/2014/02/libro-sobre-desarrollo-de-aplicaciones-con-apache-tapestry/)
-* [Prácticas de Java (2009)](https://openlibra.com/es/book/download/practicas-de-java) (PDF)
+* [PlugIn Apache Tapestry: desarrollo de aplicaciones y páginas web](https://picodotdev.github.io/blog-bitix/assets/custom/PlugInTapestry.pdf) - picodotdev (PDF) [(ePub, Mobi, HTML)](https://picodotdev.github.io/blog-bitix/2014/02/libro-sobre-desarrollo-de-aplicaciones-con-apache-tapestry/)
```
Comment (eshellman) [inline books/free-programming-books-es.md:194]: can we remake this with just one link?

File: books/free-programming-books-es.md [hunk 5]
```diff
@@ -216,44 +216,44 @@
 > :information_source: Véase también &#8230; [Angular](#angular)
 
 * [¿Cómo aprender AngularJS?](http://raulexposito.com/documentos/como-aprender-angularjs/) (HTML)
-* [AngularJS](https://eladrodriguez.gitbooks.io/angularjs) - Elad Rodriguez (HTML) (Gitbook)
-* [Guía de estilo AngularJS](https://github.com/johnpapa/angular-styleguide/blob/master/a1/i18n/es-ES.md) - John Papa, et al., `trl.:` Alberto Calleja Ríos, Gilberto (HTML)
+* [AngularJS](https://eladrodriguez.gitbooks.io/angularjs) - Elad Rodriguez (HTML) (GitBook)
```
Comment (eshellman) [inline books/free-programming-books-es.md:219]: no notation for gitbook

File: books/free-programming-books-es.md [hunk 6]
```diff
@@ -216,44 +216,44 @@
...
 
 
 ### Linux
 
 * [Bases de la Programación en Bash](http://xinfo.sourceforge.net/documentos/bash-scripting/bash-script-2.0.html) (HTML)
-* [BASH Scripting Avanzado: Utilizando Declare para definición de tipo](https://web.archive.org/web/20150307181233/http://library.originalhacker.org:80/biblioteca/articulo/ver/123) (descarga directa)
+* [BASH Scripting Avanzado: Utilizando Declare para definición de tipo](https://web.archive.org/web/20150307181233/http://library.originalhacker.org:80/biblioteca/articulo/ver/123) - Eugenia Bahit (descarga directa)
 * [El Manual de BASH Scripting Básico para Principiantes](https://es.wikibooks.org/wiki/El_Manual_de_BASH_Scripting_B%C3%A1sico_para_Principiantes) - WikiLibros
-* [El Manual del Administrador de Debian](https://debian-handbook.info/browse/es-ES/stable/) (HTML) [(PDF, ePub, Mobi)](https://debian-handbook.info/get/now/)
+* [El Manual del Administrador de Debian](https://debian-handbook.info/browse/es-ES/stable/) - Raphaël Hertzog, Roland Mas (HTML) [(PDF, ePub, Mobi)](https://debian-handbook.info/get/now/)
```
Comment (eshellman) [inline books/free-programming-books-es.md:256]: one link please

File: books/free-programming-books-es.md [hunk 7]
```diff
@@ -268,16 +268,16 @@
...
 
 
 ### NoSQL
 
 #### MongoDB
 
-* [El pequeño libro MongoDB](https://github.com/uokesita/the-little-mongodb-book)
-* [MongoDB en español: T1, El principio](https://dpdc.gitbooks.io/mongodb-en-espanol-tomo-1/content) - Yohan Graterol (Gitbook) (:construction: *en proceso*)
+* [El pequeño libro MongoDB](https://github.com/uokesita/the-little-mongodb-book) - Karl Seguin, `trl.:` Osledy Bazo
+* [MongoDB en español: T1, El principio](https://dpdc.gitbooks.io/mongodb-en-espanol-tomo-1/content) - Yohan D. Graterol (HTML) (GitBook) *(:construction: en proceso)*
```
Comment (eshellman) [inline books/free-programming-books-es.md:280]: no gitbook notation

File: books/free-programming-books-es.md [hunk 8]
```diff
@@ -336,49 +336,49 @@
...
+* [La Guía de Estilos de Ruby](https://github.com/alemohamad/ruby-style-guide/blob/master/README-esLA.md#preludio) -, Ale Mohamad (GitHub)
 * [Ruby en 20 minutos](https://www.ruby-lang.org/es/documentation/quickstart) (HTML)
-* [Ruby tutorial o cómo pasar un buen rato programando](http://rubytutorial.wikidot.com/introduccion) (HTML)
+* [Ruby tutorial o cómo pasar un buen rato programando](http://rubytutorial.wikidot.com/introduccion) - Andrés Suárez García (HTML)
 
 
 #### Ruby on Rails
 
-* [Introducción a Rails](http://rubysur.org/introduccion.a.rails/) (Gitbook)
+* [Introducción a Rails](http://rubysur.org/introduccion.a.rails/) - RubySur (HTML) (GitBook)
```
Comment (eshellman) [inline books/free-programming-books-es.md:352]: no gitbook notation

File: books/free-programming-books-es.md [hunk 9]
```diff
@@ -399,8 +399,8 @@
 
 > :information_source: Véase también &#8230; [AngularJS](#angularjs)
 
-* [Angular 2](https://rldona.gitbooks.io/angular-2-book) - Raúl López (HTML) (Gitbook)
-* [Aprendiendo Angular](https://ngchallenges.gitbook.io) - Vanessa Aristizabal (HTML) (Gitbook)
-* [Aprendizaje Angular](https://riptutorial.com/Download/angular-es.pdf) - riptutorial (PDF)
-* [Aprendizaje Angular 2](https://riptutorial.com/Download/angular-2-es.pdf) - riptutorial (PDF)
-* [Entendiendo Angular](https://jorgeucano.gitbook.io/entendiendo-angular/) - Jorge Cano (HTML, [:package: ejemplos](https://github.com/jorgeucano/entendiendo-angular)) (Gitbook)
+* [Angular 2](https://rldona.gitbooks.io/angular-2-book) - Raúl López (HTML) (GitBook)
+* [Aprendiendo Angular](https://ngchallenges.gitbook.io) - Vanessa Aristizabal (HTML) (GitBook)
```
Comment (eshellman) [inline books/free-programming-books-es.md:403]: no gitbook notation

File: books/free-programming-books-es.md [hunk 10]
```diff
@@ -399,8 +399,8 @@
...
+* [Aprendizaje Angular](https://riptutorial.com/Download/angular-es.pdf) - Compiled from StackOverflow Documentation (PDF)
+* [Aprendizaje Angular 2](https://riptutorial.com/Download/angular-2-es.pdf) - Compiled from StackOverflow Documentation (PDF)
+* [Entendiendo Angular](https://jorgeucano.gitbook.io/entendiendo-angular/) - Jorge Cano (HTML, [:package: ejemplos](https://github.com/jorgeucano/entendiendo-angular)) (GitBook)
```
Comment (eshellman) [inline books/free-programming-books-es.md:406]: no gitbook notation

**Review by eshellman — CHANGES_REQUESTED** [review]

File: books/free-programming-books-es.md (hunk 4 above)
Comment (davorpa) [reply books/free-programming-books-es.md:194]: The fact, there are only two links: one with the PDF and other with the code examples

Comment (eshellman) [reply books/free-programming-books-es.md:194]: There's no page that links to both?

Comment (davorpa) [reply books/free-programming-books-es.md:194]: Sure, but seems a blog page: https://picodotdev.github.io/blog-bitix/2014/02/libro-sobre-desarrollo-de-aplicaciones-con-apache-tapestry/

Moreover, EPUB and MOBI files there are not have link in that page like the PDF and examples:
- https://picodotdev.github.io/blog-bitix/assets/custom/PlugInTapestry.epub
- https://picodotdev.github.io/blog-bitix/assets/custom/PlugInTapestry.mobi

**Pull request merged**